dnf-behave-tests/features/microdnf/install.feature

```diff
@@ -0,0 +1,86 @@
+@no_installroot
+Feature: microdnf install command on packages with xml:base
+
+
+Background: Using repositories dnf-ci-fedora and dnf-ci-thirdparty-updates
+  Given I delete file "/etc/yum.repos.d/*.repo" with globs
+
+
+@bz1734350
+Scenario: Install package from local repodata with local packages
+#1. local repo with local packages
+Given I use repository "dnf-ci-fedora"
+ When I execute microdnf with args "install kernel"
+ Then the exit code is 0
+  And stderr is empty
+
+
+@bz1769831
+Scenario: Install package with option from local repodata with local packages
+Given I use repository "dnf-ci-fedora"
+ When I execute microdnf with args "--setopt=tsflags=nodocs install glibc"
```

Comment:
Thanks for this. Could you add another one with the more straightforward `--nodocs` and tag it with `@bz1771012`?

Also (sorry about adding more work), you should check the option did what it was meant to, which, if I understand correctly, doesn't install documentation files from RPMs. Which I think means you need an RPM with a file in `/usr/share/man/` and check it wasn't installed if this option was used.